lib/absinthe/schema/notation.ex

```diff
@@ -33,6 +33,18 @@ defmodule Absinthe.Schema.Notation do
     end
   end
 
+  defmacro topic(topic_fun) do
+    __CALLER__.module
+    |> Scope.put_attribute(:topic, topic_fun)
+    :ok
+  end
+
+  defmacro trigger(mutations, attrs) do
+    env = __CALLER__
```

Comment:
This is super minor, but these two new macros use two different syntax approaches to get the module argument into `Scope.put_attribute`. Would look nicer if they were the same.